server/src/main/java/io/seata/server/storage/redis/store/RedisTransactionStoreManager.java

```diff
@@ -494,9 +495,10 @@ private int resetLogQueryLimit(List<String> statusKeys) {
             String statusKey = buildGlobalStatus(GlobalStatus.get(param.getStatus()).getCode());
             try (Jedis jedis = JedisPooledFactory.getJedisInstance();
                  Pipeline pipelined = jedis.pipelined()) {
-                List<String> xids = pipelined.lrange(statusKey, start, end).get();
-
-                xids.stream().forEach(xid -> {
+                Response<List<String>> result = pipelined.lrange(statusKey, start, end);
+                pipelined.close();
```

Comment:
comment: get result after pipeline is closed, otherwise an error will be reported.

Reply:
get